File: solicitudAyudasClient/src/app/model/CreacionUsuarioDTO.ts
import { ComisionAprobacionUsuarioDTO } from "./ComisionAprobacionUsuarioDTO";
import { PermisoUsuarioDTO } from "./PermisoUsuarioDTO";

export class CreacionUsuarioDTO {
    public Id:number;
    public Login: string;
    public Cedula: string;
    public NombreCompleto: string;
    public SeccionalId: number;
    public Email: string;
    public Direccion: string;
    public TelefonoLabora: string;
    public TelefonoCelular: string;
    public TelefonoResidencial: string;
    public Sexo: string;
    public FechaNacimiento: Date;
    public Cargo: string;
    public Host:string;
    public PermisosUsuario:Array<PermisoUsuarioDTO> = [];
    public ComisionesAprobacion:Array<ComisionAprobacionUsuarioDTO> = [];
}

File: SolicitudAyudaServer/SolicitudAyuda.Model/DTOs/CreacionUsuarioDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SolicitudAyuda.Model.DTOs
{
    public class CreacionUsuarioDTO
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string NombreCompleto { get; set; }
        public string Login { get; set; }        
        public int SeccionalId { get; set; }
        public string Cedula { get; set; }        
        public string Seccional { get; set; }        
        public string Direccion { get; set; }
        public string TelefonoLabora { get; set; }
        public string TelefonoCelular { get; set; }
        public string TelefonoResidencial { get; set; }
        public string Sexo { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public bool Disponible { get; set; }
        public string Host { get; set; }
        public string Cargo { get; set; }
        public List<PermisoUsuarioDTO> PermisosUsuario { get; set; }
        public List<ComisionAprobacionUsuarioDTO> ComisionesAprobacion { get; set; }
    }
}
